### ISSUE TITLE
[BITV] 9.1.4.11/3.5 - In all tables, the sort status of the column is represented by an arrow symbol. This symbol does not meet the contrast requirements for graphical controls. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36973]
- [ ] F2V

![5e2cbbfe0fd4258eab94ad65371fb1fa](https://user-images.githubusercontent.com/1315170/222242719-d4464eb5-13ac-41a1-9ebc-f90abc95a7f0.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3dbfb7615a-v3-n5

</details>

- [ ] 9.1.4.11/10.5 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3dbfb7615a-v10-n5

**Comment by Pytal** [comment]
Pending completion of F2V

**Comment by Pytal** [comment]
Sortable columns on a11y instance verified

![image](https://github.com/nextcloud/server/assets/24800714/8f8a6b6c-0572-4db0-a4ab-660ea400fefa)
![image](https://github.com/nextcloud/server/assets/24800714/0c3fe664-ee65-40aa-a426-523e53e38f87)

![image](https://github.com/nextcloud/server/assets/24800714/0c1c8c6e-314c-43d9-9a44-b019baca7d8b)
![image](https://github.com/nextcloud/server/assets/24800714/d1f45ddb-fd00-41f1-87d7-098ad7f73d98)
